lineman/app/components/group_page/trial_card/donation_modal/donation_modal.coffee

```diff
@@ -0,0 +1,21 @@
+angular.module('loomioApp').factory 'DonationModal', ->
+  templateUrl: 'generated/components/group_page/trial_card/donation_modal/donation_modal.html'
+  size: 'confirm-gift-plan-modal'
+  controller: ($scope, group, $window, AppConfig, CurrentUser) ->
+    $scope.group = group
+
+    $scope.makeDonation = ->
+      $window.open "#{AppConfig.chargify.donation_url}?#{encodedChargifyParams()}", '_blank'
```

Comment:
Will want to do a check for whether AppConfig.chargify exists here (ie, whether we're on loomio.org or not)